internal/fnruntime/container.go

```diff
@@ -108,6 +108,12 @@ func (f *ContainerFn) Run(reader io.Reader, writer io.Writer) error {
 		return fmt.Errorf("unexpected function error: %w", err)
 	}
 
+	if errSink.Len() != 0 {
```

Comment:
If I understand correctly, `ContainerFn` will be wrapped by runtime.FunctionFilter in `kyaml`. I wonder if `returning error` will have some side-effect etc such as it will ignore some things considering it as a real function failure or something ?

Reply:
I understand your concern, another possible solution is adding the pointer to `FnResult` to ContainerFn struct and update the stderr directly in ContainerFn. It shares a variable across multiple different layers. Do you think it's better?

Reply:
> Do you think it's better?

Yes, I think that is much better the current approach. Ideal approach will be to update the `Run(r, w) error` interface to include function results in the output, but we don't want to go there at this stage of the project.

Reply:
done